From 11a944612ca93b8e39f6681c265c70e108f3caf2 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 24 Oct 2013 13:13:28 +0200
Subject: [PATCH] lxc-alpine: enable 4 consoles by default

We allow 4 consoles in the LXC config file so we can enable 4 in the
inittab as well.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 templates/lxc-alpine.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/templates/lxc-alpine.in b/templates/lxc-alpine.in
index 2756b89..258eed4 100644
--- a/templates/lxc-alpine.in
+++ b/templates/lxc-alpine.in
@@ -103,6 +103,9 @@ configure_alpine() {
 ::sysinit:/sbin/rc sysinit
 ::wait:/sbin/rc default
 tty1:12345:respawn:/sbin/getty 38400 tty1
+tty2:12345:respawn:/sbin/getty 38400 tty2
+tty3:12345:respawn:/sbin/getty 38400 tty3
+tty4:12345:respawn:/sbin/getty 38400 tty4
 ::ctrlaltdel:/sbin/reboot
 ::shutdown:/sbin/rc shutdown
 EOF
-- 
1.8.4.1

